### PR TITLE
misc. changes noticed during playthrough

### PR DESCRIPTION
**Pull request by cooljeanius**
Getting kind of tired of this so I might just stop here, but then again I might come back to it, so I guess I'll just leave it as a draft for now...

### CODE REVIEW

File: scenarios/16_Falling_Tears.cfg [hunk 1]
```diff
@@ -154,6 +155,7 @@
         [message]
             speaker=narrator
             image=wesnoth-icon.png
+            #po: XXX: uh... ???
```
Comment (cooljeanius) [inline scenarios/16_Falling_Tears.cfg:158]: @konrad22 opened #18 about this

File: scenarios/07_Pollur.cfg [hunk 2]
```diff
@@ -146,6 +153,7 @@
     [event]
         name=moveto
         [filter]
+            side=1
```
Comment (cooljeanius) [inline scenarios/07_Pollur.cfg:156]: this addition to the filter is for issue #6

File: scenarios/22_Fortress_Of_The_Night.cfg [hunk 3]
```diff
@@ -999,6 +997,10 @@
         [filter]
             x=9-21
             y=23-30
+            [not]
+                x=9-10
+                y=30
+            [/not]
```
Comment (cooljeanius) [inline scenarios/22_Fortress_Of_The_Night.cfg:1003]: (this part fixes #21)

**Comment by cooljeanius** [comment]
ok now that I've completed a playthrough with these changes, I'm taking it out of draft mode... if merged, this pull request:
- fixes #5 
- fixes #6 
- fixes #8 
- kinda addresses #12 a bit
- fixes #21

**Comment by cooljeanius** [comment]
@knyghtmare can you take a look please?

**Review by knyghtmare — APPROVED** [review]

**Pull request merged**